fuel-block-producer/src/adapters.rs

```diff
@@ -3,44 +3,28 @@ use crate::{
     ports::TxPool,
 };
 use fuel_core_interfaces::{
-    common::fuel_tx::{
-        CheckedTransaction,
-        ConsensusParameters,
+    model::{
+        ArcPoolTx,
+        BlockHeight,
     },
-    model::BlockHeight,
     txpool::Sender,
 };
-use std::sync::Arc;
 
 pub mod transaction_selector;
 
 pub struct TxPoolAdapter {
     pub sender: Sender,
-    pub consensus_params: ConsensusParameters,
 }
 
 #[async_trait::async_trait]
 impl TxPool for TxPoolAdapter {
     async fn get_includable_txs(
         &self,
-        block_height: BlockHeight,
+        _block_height: BlockHeight,
```

Comment:
what about block height, is it needed or ?

Reply:
It is not used right now, but it should be passed into `self.sender.includable` in the future to include only transactions valid for `block_height` .